sleap/io/visuals.py

```diff
@@ -62,7 +64,9 @@ def reader(out_q: Queue, video: Video, frames: List[int], scale: float = 1.0):
 
             # Safely load frames from video, skipping frames we can't load
             loaded_chunk_idxs, video_frame_images = video.get_frames_safely(
-                frames_idx_chunk
+                frames_idx_chunk,
+                og_video=og_video,
+                background_color=background_color
             )
```

Comment:
Instead of handling the conversion inside `Video.get_frame`, can we instead do our color correction to the `video_frame_images` numpy  returned from `Video.get_frames_safetly` - with shape (n_frames, height, width, channels)?